biosys/apps/main/tests/test_data_package.py

```diff
@@ -278,14 +278,44 @@ class TestSchemaFieldCast(TestCase):
     def setUp(self):
         self.base_field_descriptor = clone(BASE_FIELD)
 
-    def test_boolean(self):
-        true_values = [True, 'True', 'true', 'YES', 'yes', 'y', 't', '1', 1]
-        false_values = [False, 'FALSE', 'false', 'NO', 'no', 'n', 'f', '0', 0]
-        wrong_values = [2, 3, 'FLSE', 'flse', 'NON', 'oui', 'maybe', 'not sure']
+    def test_boolean_default_values(self):
+        """
+        Test possible values for a boolean
+        Since TableSchema V1.0 the default true values are [ "true", "True", "TRUE", "1" ]
+        We want to be sure that 'yes' and 'no' (and variations) are included by default.
+        """
+        true_values = ['True', 'true', 'True', 'YES', 'yes', 'y', 'Y', 'Yes']
+        false_values = ['FALSE', 'false', 'False', 'NO', 'no', 'n', 'N', 'No']
+        wrong_values = [2, 3, 'FLSE', 'flse', 'NON', 'oui', 'maybe', 'not sure', 't', '1', 1, '0', 0]
```

Comment:
Why would '0' or '1' be wrong values if they're included by default?

Reply:
I override the defaults to include 'yes' and 'no' but I also decided to exclude 0 and 1 (as discussed yesterday).
There's a problem with Travis. I'll fix it.